.github/workflows/build.yml

```diff
@@ -40,27 +40,47 @@ jobs:
         include:
           - os: ubuntu-20.04
             arch: "x86_64"
+            build: ""
```

Comment:
These lines are not necessary, but no harm either.

Reply:
I'd rather keep this verbose/explicit but if you find it makes the workflow harder to read I'm fine removing those.